src/components/PostList/PostList.tsx

```diff
@@ -10,7 +9,7 @@ import {
 import Loader from '../Loader/Loader';
 
 const PostList = () => {
-  const { data: posts, isLoading } = useGetPostsQuery(1);
+  const { data = [], isLoading } = useGetPostsQuery(1);
```

Comment:
🧐
Алиас лучше оставить
```suggestion
  const { data: posts = [], isLoading } = useGetPostsQuery(1);
```